### ISSUE TITLE
[MU4 and Muse Sound] Notes play legato when no slur is added (woodwinds but not only)

### ISSUE THREAD

**Issue by sberla365** [musescore/MuseScore#15099]
When playing clean notes without attacks or slurs, the resulting sound is legato, when the common interpretation is with a light staccato and each note is tongued. 
I attach a video with an example in which the figures sound exactly the same whether the slur is added or not. (The video is static due to an issue with my laptop, the scroll bar actually works in MU),

(The same happens with brasses and strings: clean passages and slurred play exactly the same)

https://user-images.githubusercontent.com/55401477/206876400-c904a3ec-7e92-464f-8a25-bced806f28b6.mp4

Expected behaviour: notes are tongued

**Comment by zacjansheski** [comment]
I think the lack of difference is there in both MS Basic and Muse Sounds
https://github.com/musescore/MuseScore/assets/69917893/0a6c381a-170a-4cc4-ad09-d82a370a13d4

**Comment by LucySuccubus** [comment]
These are other issue that I believe will also be fixed once this issue report is fixed as well:

#17602
#22140
#20701
#13878
#15202 (Though not specifically about legato vs. non-legato)

**Comment by wizofaus** [comment]
"light staccato" is wrong to me - I'd typically expected unslurred notes without any staccato or accent markings to be played back "sostenuto" (close to full length), but rearticulated - for winds that's tongued, for strings that's re-bowed (for percussion/keyboard instruments every note is always rearticulated, so at least for piano music you'd tend to distinguish by slightly reducing note length and compensating in "legato" by potentially slightly overlapping note lengths).

But it's also now true that depending on the sound library you use you get quite different interpretations.

**Comment by wizofaus** [comment]
BTW this is at least partly because currently the MuseSamplerSequencer only ever sends *nominal* note durations to Muse Sampler, rather than the "actualDuration" values calculated based on articulation profiles etc.

**Comment by ViolinistaInfernale** [comment]
I hear this differently. I am a violinist, but also play three other instruments from brass and woodwinds (not professionally). I hear a lightly-tongued attack on the brass and woodwinds (like a "d" sound), but a _shift_ sound between violin notes. The shift should not be between every note, only shifted notes. The sound I hear on MuseScore violin samples, like Berlin Strings, is how a one-finger scale sounds, not like changing fingers or strings (which are more like a lightly-tongued flute). There should be no slurp into a violin note unless there's a shift, and even then it will be very fast. I will add a file to this, and hope that someone will be able to get rid of this in the strings because it's unusable to me this way. I may have to go back to EastWest.

https://github.com/user-attachments/assets/3930398d-15d7-490d-b87a-1e5b02c905ff

**Comment by Fristover** [comment]
that flute is slurring, not attacking (except when there's repeated notes of course).

AFAIK, that shift on the strings is not really the same as this issue, looks more like #14524 or #13878.
You might also want to use the "Classic Phrasing" sound flag introduced to Muse Stings not too long ago.

**Comment by ViolinistaInfernale** [comment]
Hey thanks for mentioning where the Classic Phrasing lived. Hard to find in the MuseScore docs. As fas as legato is concerned, in violin it's done with separate bows but almost undetecatable bow changes. If they're slurred they are done all in one bow, just like the slur is done all in one breath with the winds. Thanks for pointing me to the other issues. I'll look at those now.

**Comment by wizofaus** [comment]
> I hear this differently. I am a violinist, but also play three other instruments from brass and woodwinds (not professionally). I hear a lightly-tongued attack on the brass and woodwinds (like a "d" sound), but a _shift_ sound between violin notes. The shift should not be between every note, only shifted notes. The sound I hear on MuseScore violin samples, like Berlin Strings, is how a one-finger scale sounds...

Yep, it seems all a "legato transition" samples for Muse Strings solo violin 1 (and to a less extent, other instruments) have an extreme amount of portamento, plus it actually sounds like there's a chorus effect on it (or like two instruments playing, slightly out of tune).  I believe the intention is to re-record those samples. 
But to me, whether there should be portamento is somewhat independent of whether a legato or non-legato sample is triggered - you can portamento between notes AND change bow (though obviously it does cause a tiny break in the sound).  And yes, portamento is something you'd generally expect when shifting position, whereas you get bizarre portamento effects when the legato samples get triggered even for passages that would surely only ever be fingered within the same hand position, e.g. written-out trills!
Fortunately Violin 2 solo isn't nearly as bad (but definitely has less "presence" to it and is less "soloistic", so not so suitable for using in a context where it's a individual soloist with other instruments in a more accompanying role).

**Comment by ViolinistaInfernale** [comment]
> But to me, whether there should be portamento is somewhat independent of whether a legato or non-legato sample is triggered - you can portamento between notes AND change bow (though obviously it does cause a tiny break in the sound).

Exactly, thank you. I can shift and change bow, effectively cutting out part of the slide. And if I play staccato, there is likely no audible portamento due to the amount of space between the notes. 

There is also some blurring in definition of articulations. All slurs are legato, but not all legato is slurring. I think a purusal of Lucan Capet's book _Superior Bowing Technique_ might be helpful, although it's a slog to get through (especially for non-string players). But the terminology confusion may be causing problems, too.

> I believe the intention is to re-record those samples.

Thank you...I had hoped for this.
